spec/views/shared/dialogs/_dialog_field_drop_down_list.html.haml_spec.rb

```diff
@@ -0,0 +1,18 @@
+describe "shared/dialogs/dialog_field_drop_down_list.html.haml" do
```

Comment:
Well, we could also keep this as a test that we actually use the helper to render the value. - but can be removed as well.